Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py

```diff
@@ -47,6 +47,48 @@ def get_incident_extra_data_by_status(incident_id, alerts_limit):
 ''' TESTS FUNCTIONS '''
 
 
+def test_get_incident_list(requests_mock):
```

Comment:
Please add docstring

Reply:
What does this test and the following one are testing? they do not use the new endpoint